go.mod

```diff
@@ -81,8 +81,6 @@ replace k8s.io/apimachinery => k8s.io/apimachinery v0.32.0
 
 replace k8s.io/apiserver => k8s.io/apiserver v0.32.0
 
-replace k8s.io/client-go => k8s.io/client-go v0.32.0
-
 replace k8s.io/component-base => k8s.io/component-base v0.32.0
```

Comment:
I think all these replace can be removed?

Reply:
Agreed

Reply:
Then why don't we just remove these?